google/cloud/storage/client.py

```diff
@@ -1089,7 +1089,7 @@ def download_blob_to_file(
             >>> from google.cloud import storage
             >>> client = storage.Client()
 
-            >>> with open('file-to-download-to', 'w') as file_obj:
+            >>> with open('file-to-download-to', 'bw') as file_obj:
```

Comment:
Thanks for the prompt fix! Just a nit, "wb" is more commonly used throughout the library and the community.

```suggestion
            >>> with open('file-to-download-to', 'wb') as file_obj:
```